### DISCUSSION TITLE
package rule to create a PR only if a specific package in the group has an update

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#29628] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

So I'm not sure if this is even possible. But let's say I have a package rule with `matchPackageNames` of A, B, and C. Would it be possible to set up the config so that a PR is created for the group only if B has an update?

For example:

A: 1.2.3->1.2.5
B: 1.0.1->_no update_
C: 2.2.2->2.3.0
_no PR created_

A: 1.2.3->_no update_
B: 1.0.1->1.3.0
C: 2.2.2->_no update_
_PR created_

A: 1.2.3->1.2.5
B: 1.0.1->1.3.0
C: 2.2.2->2.3.0
_PR created_

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
No such functionality exists
